src/apps/service/apps/de/listings.clj

```diff
@@ -488,6 +488,12 @@
     (if admin? (jobs-db/get-job-stats app-id params)
                (jobs-db/get-public-job-stats app-id params))))
 
+(defn- format-app-extra-info
+  [app-id admin?]
+  (if admin?
+    (get-app-extra-info app-id)
+    nil))
```

Comment:
This and the persistence function are both set up so that the entire `extra` key will only be included if there's something there (even an empty string, maybe), but otherwise be totally gone. We could probably change it to always include the key but default to an empty string, if that would be more useful for the UI or such, but I haven't worried about it yet.